src/submission/submission-service.ts

```diff
@@ -892,25 +896,59 @@ export namespace operations {
           .processParallel(command.clinicalType, record, index, schema);
 
         if (schemaResult.validationErrors.length > 0) {
-          let validationErrors = schemaResult.validationErrors;
+          let validationErrors: DeepReadonly<SchemaValidationError>[] = [
```

Comment:
Not an issue, but you don't need to declare the type here since it will be infered from `schemaResult.validationErrors`